Dockerfile.s390x

```
@@ -70,8 +70,11 @@ RUN --mount=type=cache,target=/root/.cache/pip \
     export PYARROW_PARALLEL=4 && \
     export ARROW_BUILD_TYPE=release && \
     python -m pip install -r requirements-build.txt && \
-    python setup.py build_ext --build-type=$ARROW_BUILD_TYPE --bundle-arrow-cpp bdist_wheel
+    python setup.py build_ext --build-type=$ARROW_BUILD_TYPE --bundle-arrow-cpp bdist_wheel && \
+    mkdir -p /tmp/arrow_wheels/ && \
+    cp dist/*.whl /tmp/arrow_wheels/
```

```suggestion
    python setup.py build_ext --build-type=$ARROW_BUILD_TYPE --bundle-arrow-cpp bdist_wheel
```

these are not necessary